src/functionaltests/resources/features/subscriptionNotification.feature

```diff
@@ -8,6 +8,7 @@ Feature: Test Subscription Trigger
     And Subscriptions are created
     When I send Eiffel events
     And Wait for EI to aggregate objects
+    Then Notification email contains <our expected> values

```

Comment:
Instead of \<our expected\> values I want you to explicitly write which values we are using and then they can be sent to the cucumber test from the feature file using REGEX. Check other feature files how this is done.